datacenter/ucp/2.1/guides/admin/configure/use-your-own-tls-certificates.md

```diff
@@ -39,11 +39,13 @@ and click **Certificates**.
 
 Upload your certificates and keys:
 
-* A ca.pem file with the root CA public certificate.
-* A cert.pem file with the TLS certificate and any intermediate CA public
+* A `ca.pem` file with the root CA public certificate followed by any intermediate CA public 
```

Comment:
Looks like you forgot to update this article for version 2.1. Once that's fixed, we're ready to ship.